index.html

```diff
@@ -0,0 +1,151 @@
+
```

Comment:
Please, I think It would be better if you add a README.md file to document your project with all important features. Please, use this [template](https://github.com/microverseinc/readme-template) but remember to customize it for your project!

Reply:
@oscar-byte15  Good job on adding inline comments, but this should be added for a specific issue in the selected line or block of code , when a file is not added you can comment about it under a required changes in the final interview message

Reply:
So, Do i need to remove my comments about missing files and put them in the final comment when i put the button review changes?